packages/dds/tree/src/feature-libraries/modular-schema/view.ts

```diff
@@ -112,14 +111,17 @@ export class ViewSchema extends ViewSchemaData<FullSchemaPolicy> {
                 fail(`tree adapter for stored ${adapter.output} should not be never`);
             }
         }
-        const adapted = new InMemoryStoredSchemaRepository(this.policy);
+        const adapted = {
```

Comment:
Did you make this change just to be a little more efficient? At the cost of a small amount of convenience? Or is there a more significant architectural reason?

Reply:
I removed the fine grained editing methods this was using before (just having one editing method simplifies the eventing) so this code had to change, and there was no reason for it to use a schema repository just to build a schema data object.